test/e2e/policies.test-suite/topology-aware/n4c16/test12-health-checking/code.var.sh

```diff
@@ -0,0 +1,25 @@
+# Test that
+# Syntatically incorrect configuration file is resulting in the nri-resmgr
+# pod liveness probe failuire.
+host-command "$SCP \"${SRC_DIR}/test/e2e/policies.test-suite/topology-aware/n4c16/test12-health-checking/nri-resmgr-configmap.yaml\" $VM_HOSTNAME:/etc/nri-resmgr/" || {
+    command-error "copying nri-resmgr-configmap.yaml to VM failed"
+}
+
+terminate nri-resmgr
+launch nri-resmgr
+vm-command "kubectl apply -f /etc/nri-resmgr/nri-resmgr-configmap.yaml"
+
+# Give enough time for the Kubernetes to consider the container unhealthy
+sleep 30
+
+# Check if we have CrashLoopBackOff status for the pod
+podstat="$(vm-command-q "kubectl get pod "$POD" -n kube-system | tail -1 | awk '{ print \$3 }'")"
```

Comment:
Where is the `POD` variable coming from?
Couldn't we just call `vm-wait-pod-regexp` here?

Reply:
It is coming from https://github.com/jukkar/nri-resmgr/blob/b92ed0f23a5461effde8b8718e61a51b0da58051/test/e2e/run.sh#L246.

Reply:
Hmm, does this test really pass? I would have thought that the POD variable is not seen in this shell script.